orbit/models/dlt.py

```diff
@@ -145,39 +146,16 @@ def _set_init_values(self):
         See: https://pystan.readthedocs.io/en/latest/api.htm
         Overriding :func: `~orbit.models.BaseETS._set_init_values`
         """
-        def init_values_function(s, n_pr, n_nr, n_rr):
-            init_values = dict()
-            if s > 1:
-                init_sea = np.random.normal(loc=0, scale=0.05, size=s - 1)
-                # catch cases with extreme values
-                init_sea[init_sea > 1.0] = 1.0
-                init_sea[init_sea < -1.0] = -1.0
-                init_values['init_sea'] = init_sea
-            if n_pr > 0:
-                x = np.random.normal(loc=0, scale=0.1, size=n_pr)
-                x[x < 0] = -1 * x[x < 0]
-                init_values['pr_beta'] = x
-            if n_nr > 0:
-                x = np.random.normal(loc=-0, scale=0.1, size=n_nr)
-                x[x > 0] = -1 * x[x > 0]
-                init_values['nr_beta'] = x
-            if n_rr > 0:
-                init_values['rr_beta'] = np.random.normal(loc=-0, scale=0.1, size=n_rr)
-            return init_values
-
-        seasonality = self._seasonality
-
         # init_values_partial = partial(init_values_callable, seasonality=seasonality)
         # partialfunc does not work when passed to PyStan because PyStan uses
         # inspect.getargspec(func) which seems to raise an exception with keyword-only args
         # caused by using partialfunc
         # lambda as an alternative workaround
-        if seasonality > 1 or self._num_of_regressors > 0:
-            init_values_callable = lambda: init_values_function(
-                seasonality,
-                self._num_of_positive_regressors,
-                self._num_of_negative_regressors,
-                self._num_of_regular_regressors)
+        if self._seasonality > 1 or self._num_of_regressors > 0:
```

Comment:
This is a clever work around! I'm seeing my notes now on why we couldn't use `partialfunc` in the first place and why the workarounds are needed. Perhaps later we can also contribute to PyStan upstream to resolve the issue with `partialfunc`